bundle/permissions/workspace_root.go

```diff
@@ -18,6 +19,12 @@ func ApplyWorkspaceRootPermissions() bundle.Mutator {
 
 // Apply implements bundle.Mutator.
 func (*workspaceRootPermissions) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
+
+	diags := checkWorkspaceRootPermissions(b)
+	if len(diags) > 0 {
+		return diags
```

Comment:
Why return?

Reply:
OK, that works. Please add a comment to explain what is happening. E.g. the diag only contains a warning, so subsequent mutators will still run. This is not clear from the code as is.

Alternatively, you can make the function return a bool to indicate if the bundle root is under `/Shared`, and only run the permission routine if that is false.